README.md

```diff
@@ -1,27 +1,66 @@
 # c2st
 
-## two sample test using a ML classifier
+## Two sample test using a ML classifier
 
-``` python
-> from c2st.check import c2st as compare
+Test whether sets of D-dimensional points are samples from the same
+multivariate probability distribution.
 
-#let's say you have 2 samples X and Y and want to know 
-#if they come from the same distribution
+The `c2st` function returns the accuracy of how well a binary classifier was
+able to classify two sets of points `X` and `Y` while being trained on the
+concatenated dataset `(X,Y)`. All samples of `X` have received the label `0`
+and `Y` has received the label `1`.
 
-> compare(X,Y)
-[0.51904464] #X and Y are likely from the same PDF
-```
+A value close to 0.5 means that the classifier is not better than random
+guessing, i.e. `X` and `Y` are likely from the same distribution. A value close
+to 1 means the classifier was able to separate `X` and `Y`, so they are
+probably samples from different distributions.
+
+
+```py
+>>> import numpy as np
+>>> from c2st.check import c2st
+
+>>> rng=np.random.default_rng(seed=123)
+
+# same distribution (Gaussian N(0,1)), D=20, 1000 points each
+>>> X=rng.normal(loc=0, scale=1, size=(1000,20))
+>>> Y=rng.normal(loc=0, scale=1, size=(1000,20))
+>>> c2st(X, Y)
+0.4970122828225085
 
-The `compare` method returns (by default) the accuracy on how well a binary classifyer was able to classify Y from X while being trained on the concatenated dataset `(X,Y)`. All samples of `X` have received the label `0` and `Y` has received the label `1`.
+# now shift the mean of Y a bit
```

Comment:
```suggestion
# now shift the mean of Y by `0.3`
```

Reply:
Let's be precise and help people not familiar with the `np.random` API to understand the example.